frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -490,10 +490,15 @@ const GitSection: React.FC<GitSectionProps> = ({
         helpText={helpText}
         helpTextInvalid={helpText}
         validated={validated}
+        onBlur={(e: React.SyntheticEvent) => {
+          const trimmedURL = (e.target as HTMLInputElement).value.trim();
+          setFieldValue('git.url', trimmedURL);
+          debouncedHandleGitUrlChange(trimmedURL, values.git.ref, values.git.dir);
```

Comment:
Is it possible to skip the debounce here? When the user leaves the field, we don't need to wait n ms, right? But I don't know if this possible because the onChange will maybe then trigger it again... It's just an idea, if there is no easy solution we can keep the debounce here for the moment.

Reply:
I tested with skipping the debounce. 
But then, after the URL gets trimmed, the Git URL validation won't run. Only after the validation passes, does the `Create` button become active. To validate in this case, the user will have to rely on the `onChange` event's debounce function for validation. 
Thus skipping the debounce causes validation issues in the `onBlur` function for now.